### PR TITLE
Deploy in Padlocal

### PR DESCRIPTION
**Pull request by abhishek-iiit**
## Deploy in PadLocal
Screenshot from 2021-07-29 20-25-02
Screenshot from 2021-07-29 20-25-10
Screenshot from 2021-07-29 20-25-21

### CODE REVIEW

**Comment by abhishek-iiit** [comment]
Linked to issue #896

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by vasvi-sood — APPROVED** [review]

File: docusaurus/docs/howto/deploy-padlocal.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+![Deploy in PadLocal](../../static/img/docs/howto/IM_platform/PadLocal.webp)
+
+## Requirements
+
+1. Your system must have [Node.js](https://nodejs.org/en/download/package-manager/) installed (version >= 12).
+2. Your system must have [Wechaty](https://github.com/wechaty/wechaty) (version >= 0.40).
+3. You must be familiar with [Wechaty puppet padlocal](https://www.npmjs.com/package/wechaty-puppet-padlocal).
+
+## Deployment
+
```
Comment (proudofsimin) [inline docusaurus/docs/howto/deploy-padlocal.mdx:16]: Run `wechaty-puppet-padlocal`:

**Review by proudofsimin — APPROVED** [review]
Thank you.

File: docusaurus/docs/howto/deploy-padlocal.mdx (hunk 1 above)
Comment (abhishek-iiit) [reply docusaurus/docs/howto/deploy-padlocal.mdx:16]: Done as suggested :)

**Review by huan — APPROVED** [review]
LGTM.

```diff
- docusaurus/static/img/docs/howto/IM_platform/PadLocal.webp 
+ docusaurus/static/img/docs/howto/deploy-padlocal/padlocal.webp 
```

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Comment by huan** [comment]
@shraddhavp Please notice my last comment about the filename rename request.

You can create a new PR to fix that, thanks.

**Comment by shraddhavp** [comment]
@abhishek-iiit

**Comment by shraddhavp** [comment]
> @shraddhavp Please notice my last comment about the filename rename request.
> 
> You can create a new PR to fix that, thanks.

yes sure @huan

**Comment by abhishek-iiit** [comment]
Sure will change the same :)  @huan @shraddhavp

**Pull request merged**